lib/wca_live/synchronization/export.ex

```diff
@@ -13,6 +13,12 @@ defmodule WcaLive.Synchronization.Export do
     |> competition_to_wcif()
   end
 
+  def export_results(competition) do
```

Comment:
The `Synchronization.{Import,Export}` modules are specifically about WCIF and I wouldn't mix the responsibilities.

I think it's actually fine to build the response in the controller (normally we would have a separate "view" modules for JSON responses, but for a single endpoint either way is fine).

Reply:
I can refactor for this.